lib/src/atom/matcher.rs

```diff
@@ -428,7 +428,9 @@ impl Bindings {
             });
 
         let results = results.into_iter().map(|(result, _)| result).collect();
-        log::trace!("Bindings::merge: {} ^ {} -> {:?}", trace_self.unwrap(), b, results);
+        if let Some(self_copy) = trace_self {
```

Comment:
Ah.  Never mind.  I didn't test the case where tracing was enabled for some modules and not others.

Reply:
> QUESTION: Did this unwrap cause a panic, or is this change mainly for style?

It was a surprise to me, because I though the `log::trace!` macro calculates arguments after the log's level is checked. But in a case when tracing is enabled for a single module for instance, this check happens after calculating of the arguments. `log` crate has an issue on this: https://github.com/rust-lang/log/issues/472. The comment https://github.com/rust-lang/log/issues/472#issuecomment-974190821 explains the reason. This behavior was introduced to be a logging "facade".